README.md

```diff
@@ -95,7 +94,7 @@ For MNI and DICOM tools:
 | Mnc2nii       | To run mnc2nii, in order to convert MINC to NIfTI                                              |
 | Nii2mnc       | To run nii2mnc, in order to convert NIfTI to MINC                                              |
 | NuCorrect     | To run nu_correct                                                                              |
-
+| MincBet       | To run brain extraction tool wrapper                                                           |
```

Comment:
Could add that it requires https://github.com/aces/cbrain/issues/373 to be fixed, just as a reminder.